### PR TITLE
October comes early! Revolutionary art and bust.

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request
This PR adds a new art component that gives a good moodlet to revolutionaries. I've also added a bronze bust of the father of sociology, Karl Marx to illustrate this functionality. Inspired by my trip to the Stasi headquarters in Berlin.

## Why It's Good For The Game
Rev gameplay is always more fun if you do some light RP such as yelling communist slogans. It can also be used for Russian station RP and more!.

## Changelog
:cl:
add: You can now craft a bronze bust depicting economist Karl Marx.
/:cl:

MarxPR

### CODE REVIEW

**Comment by Dawson1917** [comment]
POLITICAL AGENDAS IN MY VIDEOGAMES

POLITICAL AGENDAS IN MY VIDEOGAMES

**Comment by FancyCyclops** [comment]
new meta: show suspects pictures of karl marx

if they look happier, chop their heads off

**Comment by Krysonism** [comment]
This is actually a statue at the moment, not a handheld bust. If this PR fails I will try again in item form.

As for the ideology of the revs, I don't think it has been stated anywhere, I think them being communists is both funnier and is more culturally relevant given how much communist revolutions and the fear of them has shaped the world in the 20th century.

**Comment by FantasticFwoosh** [comment]
Kitchen's closed 👨🏾‍✋🏾 , Kitchen's closed 👨🏾‍✋🏾 , Kitchen's closed 👨🏾‍✋🏾

File: code/datums/components/art.dm [hunk 1]
```diff
@@ -42,6 +42,17 @@
...
 	on_obj_examine(source, M)
 
+/datum/component/art/rev
+
+/datum/component/art/rev/apply_moodlet(mob/M, impress)
+	M.visible_message("<span class='notice'>[M] stops to inspect [parent].</span>", \
+						 "<span class='notice'>You take in [parent], inspecting the fine craftsmanship of the proletariat.</span>")
+
+	if(M.mind && M.mind.has_antag_datum(/datum/antagonist/rev))
+		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artgreat", /datum/mood_event/artgreat)
```
[the remark this answers is not in the document]
Comment (Krysonism) [reply code/datums/components/art.dm]: Can living players see mood? It seems like you would rather watch for people getting flashed or beating the shit out of heads if you are ghosted and wanted to metagame revs.

**Comment by Krysonism** [comment]
> 
> 
> ok, but can you wield it like a mace and bash the capitalist bourgeoisie?
> 
> image

Only if i decide to make it a held item, maybe i could make it an item that turns into a structure if wrenched.

**Comment by Krysonism** [comment]
Would you guys rather have this as a reskin of the other busts?

 Would a 15 force blunt weapon craftable from 50-25 bronze sheets be overpowered?

**Comment by Dawson1917** [comment]
heads of staff should get get a fearful moodlet on examining one

**Comment by daklj** [comment]
mind telling how long did you work on the sprite?

**Comment by Krysonism** [comment]
>mind telling how long did you work on the sprite?

Maybe an hour and a half to two hours.

 I drew it In my hotel room in Berlin after visiting the stasi headquarters which is now a museum. They had several busts, mosaics and other artwork depicting Marx, Lenin and Dzerzhinsky, more than you might expect. I felt very inspired to sprite something and this felt like this would be a fun exercise.

I had to make some adjustments to the beard and hair once I came home and I looked at the sprite with a fresh set of eyes.

It went much more smoothly than expected, which I guess because I was in kind of a strange mood at the time. 

I am pretty happy with the results, although one criticism i have is that it has a lot more contrast than many sprites in the game, which can make it look a bit out of place around low contrast sprites.

I recommend a visit by the way if you ever go to Berlin.

File: code/datums/components/art.dm [hunk 2]
```diff
@@ -42,6 +42,17 @@
 		return
 	on_obj_examine(source, M)
 
+/datum/component/art/rev
+
+/datum/component/art/rev/apply_moodlet(mob/M, impress)
+	M.visible_message("<span class='notice'>[M] stops to inspect [parent].</span>", \
+						 "<span class='notice'>You take in [parent], inspecting the fine craftsmanship of the proletariat.</span>")
```
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/datums/components/art.dm]: traitortest potential but then again it's revs so 🤷‍♂

File: code/datums/components/art.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (ExcessiveUseOfCobblestone) [reply code/datums/components/art.dm]: I think there are ways (mindreader if thats in) that allows mood reading of some sort.

Like I said above though, it's revs. If you have time to traitor test you're either about to lose hard or are steamrolling.

**Comment by AnturK** [comment]
Merge conflicts.

**Comment by AnturK** [comment]
Hello ? Runtimes on unknown undefs still.

**Comment by Krysonism** [comment]
Runtimies? I will fire this up tonight and try to sort it out.

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 3]
```diff
@@ -571,7 +571,8 @@ GLOBAL_LIST_INIT(bronze_recipes, list ( \
 	new/datum/stack_recipe("bronze boots", /obj/item/clothing/shoes/bronze), \
 	null,
 	new/datum/stack_recipe("bronze chair", /obj/structure/chair/bronze, 1, time = 0, one_per_turf = TRUE, on_floor = TRUE), \
-))
+	new/datum/stack_recipe("Marx Bust", /obj/structure/statue/bronze/marx, 15, one_per_turf = 1, on_floor = 1), \
+))f
```
Comment (AnturK) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: `f` something went wrong here.

Comment (Krysonism) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: f

**Pull request merged**